gradle.properties

```diff
@@ -12,13 +12,13 @@ ballerinaGradlePluginVersion=1.0.0
 ballerinaLangVersion=2201.5.0
 
 #stdlib dependencies
-stdlibIoVersion=1.4.0
-stdlibLogVersion=2.7.0
-stdlibFileVersion=1.6.0
+stdlibIoVersion=1.4.1
+stdlibLogVersion=2.7.1-20230522-155500-d73bdb5
+stdlibFileVersion=1.7.1-20230523-104800-e56ed72
```

Comment:
Better to bump the module to the next minor version as this dependency is a minor version bump

Reply:
Updated in [2f1909f](https://github.com/ballerina-platform/module-ballerina-yaml/pull/9/commits/2f1909fe4cf21431ebb096406fb73e18e0c91d29)